### ISSUE TITLE
Broken Manuels

### ISSUE THREAD

**Issue by Aligotess13** [ParadiseSS13/Paradise#17573]
**Issue Description**:
When using a manual, the wiki page it opens up to has extreme formatting errors and looks really cramped. You can't use mouse scrolling either.

**What did you expect to happen**: 
For the wiki page to open as it would if you normally minimize the screen. 

**What happened instead**:
Titles were mixing into each other, the mouse scroll became unusable, and all content was pre-expanded.

**Why is this bad/What are the consequences**:
It makes manuals less useful, especially as an IC method of educating new players.

**Steps to reproduce the problem**:
Opening up a manual.

**When did the problem start happening**:
I believe it was an issue ever since I started playing, so I wouldn't know when it began.

**Extra information**:
I'm sure this may be an open secret since I've read that in the past, these books would just show a blank screen. However, I still believe it's an issue that may/should be looked into in the future.

**Comment by Sirryan2002** [comment]
tl;dr MediaWiki used to have a "printable" version mode for wiki pages that could be enabled via variables in links which our in-game manuals still use through iframes. However, this was depreciated in newer versions of mediawiki https://phabricator.wikimedia.org/T167956

It is likely that this will not be an easy fix b/c a custom css stylesheet (with iframe mediaqueries) or some java/jquery magic needs to be used in order to fix this. I will look into possible solutions.

**Comment by Sirryan2002** [comment]
update, found an easy fix to it, should be fixed in library overhaul PR
